terraform/variables.tf

```diff
@@ -98,4 +98,9 @@ variable "lb_port" {
 variable "hd_insight_count" {
   # This is added due to resource constraints.
   default = 0
+}
+
+variable "public_ip_address_count" {
+  # This is added due to resource constraints.
+  default = 0
```

Comment:
Let's make this default `1`, and see how the integration tests go.

Reply:
done